dev-tools/compose/docker-compose.storagev2.yml

```diff
@@ -43,6 +43,6 @@ services:
           'curl --fail http://localhost:1991/health_check || exit 1'
         ]
       interval: 10s
-      start_period: 30s
+      start_period: 600s
```

Comment:
why does this need to be so high? i would think if the migration is removed then there's not as much to wait for before it's healthy

Reply:
changed to 60s. should take less than 30s to migrate all the partition tables but bumping it just to be safe. the 600s was from the previous PR but after testing the new sql structure more 60s seems to be sufficient

Reply:
nice that sounds good